src/points/PointsHistoryBottomSheet.test.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import PointsHistoryBottomSheet from 'src/points/PointsHistoryBottomSheet'
+import { createMockStore, RecursivePartial } from 'test/utils'
+import { RootState } from 'src/redux/reducers'
+import { FetchMock } from 'jest-fetch-mock/types'
+import { Provider } from 'react-redux'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { PointsEvents } from 'src/analytics/Events'
+import { getHistoryStarted } from 'src/points/slice'
+import { GetHistoryResponse } from 'src/points/types'
+
+jest.mock('@gorhom/bottom-sheet', () => {
+  const react = require('react-native')
+  return {
+    __esModule: true,
+    default: react.View,
+    BottomSheetSectionList: react.SectionList,
+  }
+})
+
+const MOCK_RESPONSE_NO_NEXT_PAGE: GetHistoryResponse = {
+  data: [
+    {
+      activity: 'swap',
+      points: '20',
+      createdAt: '2024-03-05T19:26:25.000Z',
+      metadata: {
+        to: 'celo-alfajores:native',
+        from: 'celo-alfajores:0x874069fa1eb16d44d622f2e0ca25eea172369bc1',
+      },
+    },
+    {
+      activity: 'swap',
+      points: '20',
+      createdAt: '2024-01-04T19:26:25.000Z',
+      metadata: {
+        to: 'celo-alfajores:0xe4d517785d091d3c54818832db6094bcc2744545',
+        from: 'celo-alfajores:native',
+      },
+    },
+  ],
+  hasNextPage: false,
+  nextPageUrl: '',
+}
+
+describe(PointsHistoryBottomSheet, () => {
+  const mockFetch = fetch as FetchMock
+  beforeEach(() => {
+    jest.useRealTimers()
```

Comment:
why use real timers?

Reply:
whoooops, don't think that belongs here :)